martin/src/cog/source.rs

```diff
@@ -419,33 +289,22 @@ fn get_image_dims(
 
     Ok((image_width, image_length))
 }
+fn dim_in_model(
```

Comment:
Could you add add a doc comment explaining what this does? I missed something or don't have in my head anymore what this is.